source/refills/tooltip.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href="#0" class="tooltipped" aria-label="Tooltip description">
```

Comment:
Quick check with VoiceOver: when focus hits this element, the screen only announces the content within `aria-label` and disregards anything within the anchor itself.

Screen recording of this: https://www.dropbox.com/s/aux5aepyjzu8i7c/tooltip-with-voiceover.mov?dl=0